beancount_import/matching.py

```diff
@@ -240,6 +254,20 @@ def get_fuzzy_date_range(self, orig_date: datetime.date):
                                 self.fuzzy_match_days + 1):
             yield orig_date + datetime.timedelta(days=day_offset)
 
+    def fuzz_date_currency_key(self, key: Tuple):
```

Comment:
Type annotation should be `DateCurrencyKey` rather than `Tuple`, I think

Reply:
Also please add return annotation, e.g. `Iterator[DateCurrencyKey]`

Reply:
Done.